tcms/rpc/tests/test_testexecution.py

```diff
@@ -235,6 +235,29 @@ def test_detach_log(self):
         self.assertEqual([], list(self.case_run.links()))
 
 
+class TestExecutionRemoveLinkPermissions(APIPermissionsTestCase):
+    """Test permissions of TestExecution.remove_link"""
+
+    permission_label = 'linkreference.delete_linkreference'
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.case_run = TestExecutionFactory()
+        self.link = LinkReferenceFactory(execution=self.case_run)
+        self.another_link = LinkReferenceFactory(execution=self.case_run)
+
+    def verify_api_with_permission(self):
+        self.rpc_client.TestExecution.remove_link({'pk':  self.link.pk})
+        links = self.case_run.links()
+        self.assertEqual(1, links.count())
```

Comment:
start by asserting that there are 2 URLs linked to this TE before deleting one of them